proxy/mempool/mempool_schedule.py

```diff
@@ -274,7 +275,7 @@ def __init__(self, capacity: int, chain_id: int) -> None:
 
     @property
     def min_gas_price(self) -> int:
-        if self.tx_cnt < int(self._capacity * 0.9):  # if there are more than 90%
+        if self.tx_cnt < int(self._capacity * self._capacity_high_watermark):
```

Comment:
It is just a comment, not a flaw.
You calculate the same value in two places without any difference.
So, the value can be calculated just one time and used in both places

Reply:
Yep, sounds reasonable. I've updated to pre-calculate and allow it to be changed publicly.